.gitignore

```diff
@@ -62,3 +62,5 @@ typings/
 
 # next.js build output
 .next
+
+.vscode
```

Comment:
shouldn't this belong in your `.git/info/exclude`?

Reply:
fixed!